Data/Dto/PostDto.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+
+namespace Blogger.Data.Dto
+{
+    public class PostDto : IRequestDto<Post>
+    {
+
+        public string Title {set; get;}
+        public string Body {set; get;}
+        public DateTime CreatedOn {set; get;}
+        
+        public dynamic Errors = null;
```

Comment:
How would it be if we can support list of errors? Why not a string instead of dynamic?
How about having the error object in some base class/some magic box?

Reply:
I used `dynamic` as for creating anonymous object as like `JSON` format.

Reply:
will be added on next commit